TheSadRogue.Primitives/Point.cs

```diff
@@ -546,6 +546,34 @@ public PolarCoordinate ToPolarCoordinate()
         {
             return PolarCoordinate.FromCartesian(this);
         }
+
+        /// <summary>
+        /// Rotates a single point around the origin (0, 0).
+        /// </summary>
+        /// <param name="degrees">The amount of Degrees to rotate this point clockwise</param>
+        /// <returns>The equivalent point after a rotation</returns>
+        public  Point Rotate(in double degrees)
+        {
+            double radians = MathHelpers.DegreesToRadiansConstant * degrees;
```

Comment:
Provided the `MathHelpers` fix is made, why not:
```suggestion
            double radians = MathHelpers.ToRadian(degrees);
```